modules/admin-limit-operations.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// ../scalability_and_performance/compute-resource-quotas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="admin-limit-operations_{context}"]
+= Limit range operations
+
+== Creating a limit range
+
+To apply a limit range to a project, create a limit range object definition with the required specifications.
+
+. Create a limit range object definition with your required specifications.
+
+.Procedure
+
+. Create the object:
```

Comment:
```suggestion
2 . Create the object:
```

Reply:
The "." is an auto-numbering feature in asciidoctor, so I do not have to put a number on the line.